src/lib/free-queue/test.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>FreeQueueSAB Tests</title>
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/mocha/10.6.0/mocha.min.css" integrity="sha384-BBZj3VfQ5pAok2bd0uXf+8cRAJ3D/JZWrxBlnWx6BcsBO96ZzZR40fpC20MtWbM8" crossorigin="anonymous">
```

Comment:
When I tried to test this locally, I got an error that the hash didn't match.

I think we can just copy the SHA-512 hash directly from cdnjs:
https://cdnjs.com/libraries/mocha
(click the shield icon next to the filename)